lib/openh264.py

```diff
@@ -41,9 +41,10 @@ def EncodeCommandLine(self, parameters, bitrate, videofile, encodedfile):
         '-bf %s '
         '-sw %d -sh %d '
         '-dw 0 %d -dh 0 %d '
-        '-rc 0 -maxbrTotal %d -tarb %d '
+        '-rc 0 -maxbrTotal 0 -tarb %d '
         '-ltarb 0 %d '
-        '-lmaxb 0 %d '
+        '-lmaxb 0 0 '
+        '-fs 0' #disable FrameSkip with this option
```

Comment:
At the moment, frame skip is disabled by editing the config file (see compile_tools.sh). With this command line parameter, can we delete the "edit the config file" hack?